### ISSUE TITLE
Exclude .terragrunt-cache path from indexing

### ISSUE THREAD

**Issue by khartahk** [hashicorp/vscode-terraform#169]
I'm using terragrunt with terraform and it creates a lot of copies of *.tf files under .terragrunt-cache folder. Those are then being indexed and consume a lot of CPU as the projects are quite large. How can I exclude the path from being indexed?

**Comment by mauve** [comment]
there is a setting for that called something like
`terraform.indexing.exclude` or something like that

**Comment by khartahk** [comment]
Thank you. I didn't know that copying the json config string in VS Code could hold an array of settings. I've copied indexing and I got the entire object:
```
"terraform.indexing": {
    "enabled": true,
    "liveIndexing": true,
    "delay": 500,
    "exclude": [
        ".terraform/**/*",
        "**/.terraform/**/*",
        ".terragrunt-cache/**/*",
        "**/.terragrunt-cache/**/*"
    ]
}
```
It would be nice if all the config options were available in the documentation or it was mentioned you can copy the json object in VS Code settings for the extension :)

**Comment by khartahk** [comment]
No matter what I put in it doesn't exclude anything. Even If I put `*` it still indexes everything.

**Comment by mauve** [comment]
which version do you have of the plugin? a recent release fixed a bug
which consumed CPU even for ignored files.

**Comment by khartahk** [comment]
Version 1.3.9

**Comment by udondan** [comment]
Thanks @khartahk for posting the exclude patterns. That works! Had to add this to a couple of other plugins too. My VSC was running hot whenever I starteed terragrunt.

**Comment by paultyng** [comment]
We just released [v2.0.0-rc.1](https://github.com/hashicorp/vscode-terraform/releases/tag/v2.0.0-rc.1) of the extension. The main features include:

* Added syntax support for 0.12
* Added [terraform-ls](https://github.com/hashicorp/terraform-ls) usage by default (currently on 0.3.0, which offers basic provider code completion)

You can find additional information and specifics in the [release notes](https://github.com/hashicorp/vscode-terraform/releases/tag/v2.0.0-rc.1) and [CHANGELOG](https://github.com/hashicorp/vscode-terraform/blob/master/CHANGELOG.md).

With this release we expect that many of the prior issues and PRs are no longer relevant or have been addressed, and are therefore being closed. If you feel the action taken on an issue or PR is in error, please comment as such and we can figure out the appropriate way to address it.

We plan to add the final 2.0.0 release to the marketplace soon, but are actively seeking your feedback now on the release candidates. You can download the .vsix from the releases page and [manually install it in VS Code](https://code.visualstudio.com/docs/editor/extension-gallery#_install-from-a-vsix) to try it out.
